### PR TITLE
Show testing userIds for multi-pref-experiment branches

### PR DESCRIPTION
**Pull request by mythmon**
I've been sitting on this for a while, working on it in the background. These testing IDs are important to the recent QA process of testing specific branches of experiments, and having them easily accessible is very nice.

The testing ID shows up for each branch:

image

---

And also generally for the experiment (as it did before):

image

### CODE REVIEW

File: content/components/recipes/details/arguments/MultiPreferenceExperiment.js [hunk 1]
```diff
@@ -85,6 +90,29 @@ export default function MultiPreferenceExperiment({ data }) {
                       <code>{branch.ratio}</code>
                     </div>
                   </div>
+                  <div className="flex-basis-0 flex-grow-1">
+                    <div className="d-flex align-items-center">
+                      <div className="pr-2 font-weight-bold">
+                        Testing clientId
+                        <HelpIcon>
+                          Setting the preference{" "}
```
Comment (rehandalal) [inline content/components/recipes/details/arguments/MultiPreferenceExperiment.js:98]: nit: i found this a little confusing `{" "}`. It should probably be `&nbsp;`

**Review by rehandalal — CHANGES_REQUESTED** [review]
r+wc

File: content/components/recipes/details/arguments/MultiPreferenceExperiment.js (hunk 1 above)
Comment (mythmon) [reply content/components/recipes/details/arguments/MultiPreferenceExperiment.js:98]: This is how prettier formats text when it wraps JSX lines. I tested and Prettier will leave `&nbsp;` if I put it there, but I think the automatic thing is easier.

Comment (rehandalal) [reply content/components/recipes/details/arguments/MultiPreferenceExperiment.js:98]: oh right, this is all coming back to me.

**Review by rehandalal — APPROVED** [review]
bors r+

**Pull request merged**